apps/src/tests/single-feature-tests/stack-v5/test-stack-subviews-ios/scenario.md

```
@@ -81,12 +82,21 @@ stay manual.
 12. Set title to `long` and `largeTitle` to `short`. Scroll to reveal large title.
 
   - [ ] on iOS 18, largeTitle should be long
+  
   - [ ] on iOS 26, largeTitle should be short
 
 13. Set `largeTitle` to `none`.
 
   - [ ] on iOS 26, largeTitle should be long
 
+  ## hidesSharedBackground (iOS 26)
```

markdownlint-cli2 (0.23.2)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)